docs/mods/suite/Quests/Developer/Integrations/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Integrations
+draft: true
+---
+
+FTB Quests has a number of integrations with other mods. These integrations allow you to use the features of other mods in your quests.
+
+:::caution
+WIP
+:::
+
+FTB Filter system but that’s a mod that needs it’s own docs anyway. Basically, complex filters that allows authors to configure item checks via a GUI
+
+JEI, REI: Works with lib as well and shows a dedicated page on the item selection for JEI items, it also adds quest items as a reward section in the uses / recipe tabs, it might do more, not sure.
```

Comment:
Also loot crates - if a reward table has a loot crate, then JEI/REI will show what items drop from that loot crate.